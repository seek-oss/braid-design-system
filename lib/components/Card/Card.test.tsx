import React from 'react';
import { Card } from '..';
import { render } from 'react-testing-library';
import forEachTheme from '../../test/utils/forEachTheme';

describe('Card', () => {
  forEachTheme(({ ThemeProvider }) => {
    test('Default', () => {
      const { container } = render(
        <ThemeProvider>
          <Card>Children</Card>
        </ThemeProvider>
      );
      expect(container.firstChild).toMatchSnapshot();
    });
  });
});
