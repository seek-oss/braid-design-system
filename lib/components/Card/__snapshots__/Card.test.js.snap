// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Theme: jobStreet Default 1`] = `
<div
  class="Box__root backgroundColor__card borderColor__standard marginBottom__medium"
>
  Children
</div>
`;

exports[`Card Theme: seekAnz Default 1`] = `
<div
  class="Box__root backgroundColor__card borderColor__standard marginBottom__medium"
>
  Children
</div>
`;

exports[`Card Theme: seekAsia Default 1`] = `
<div
  class="Box__root backgroundColor__card borderColor__standard marginBottom__medium"
>
  Children
</div>
`;

exports[`Card Theme: wireframe Default 1`] = `
<div
  class="Box__root backgroundColor__card borderColor__standard marginBottom__medium"
>
  Children
</div>
`;
